packages/babel-helpers/src/helpers/applyDecs2305.js

```diff
@@ -319,7 +319,7 @@ function applyMemberDec(
       init = function (instance, init) {
         var value = init;
 
-        for (var i = 0; i < ownInitializers.length; i++) {
+        for (var i = ownInitializers.length - 1; i >= 0; i--) {
```

Comment:
If a decorator calls `context.addInitializer()` twice:
```js
context.addInitializer(f);
context.addInitializer(g);
```
Previously `f` will be invoked before `g` when the class member is initialized, now `f` will be invoked _after_ `g`. @pzuraq is that right?

Reply:
No, the order of `addInitializer` initializers has not been changed, only initializers returned from `init`